apps/example/ios/Example.xcodeproj/project.pbxproj

```diff
@@ -31,7 +30,6 @@
 		00E356EE1AD99517003FC87E /* ExampleTests.xctest */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; path = ExampleTests.xctest; sourceTree = BUILT_PRODUCTS_DIR; };
 		00E356F11AD99517003FC87E /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		00E356F21AD99517003FC87E /* ExampleTests.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = ExampleTests.m; sourceTree = "<group>"; };
-		0A814516296EE5C500B3A131 /* GoogleService-Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = "GoogleService-Info.plist"; sourceTree = "<group>"; };
```

Comment:
dont we need plist inresources so it can compile?

file should be added in CI pipelne from secrets already

Reply:
Are we sure `.plist` files are added from the CI pipeline, as 5 days ago this [Build & Code Quality Checks](https://github.com/rudderlabs/rudder-sdk-react-native/actions/runs/3903848579/jobs/6668794483#logs) test failed.
This failures happens when `.plist` file are not present.

Reply:
We have a step Add GoogleService-Info.plist. If this fails we need to fix it

Reply:
We need to fix the step which adds `GoogleService-Info.plist` file.
CI tests are failing: https://github.com/rudderlabs/rudder-sdk-react-native/actions/runs/3947543214/jobs/6756479769

Reply:
fixed